docs/force_rescans.md

```diff
@@ -0,0 +1,81 @@
+# Rebuilding transaction history
+
+It is unlikely, but possible and unfortunate, that transaction history in the
+wallet database may not represent reality.  This may be due to a programming
+mistake or the transaction database becoming corrupted.  Thankfully, all
+transactions are publicly recorded on the blockchain, and transactions
+necessary for a fully functional wallet can be recovered.  This process is
+called rescanning, and the following guide will demonstrate how to force such a
+rescan.
+
+Rescans are automatically performed each time the wallet syncs to the network.
+These are used to "catch up" the wallet to the newest best block in the block
+chain.  For example, the following log messages at startup indicate that an
+out-of-sync wallet started a rescan for all addresses and unspent outputs since
+some block.
+
+```
+13:45:03 2015-04-13 [INF] WLLT: Started rescan from block 00000000001703b1a9dfd4865d587cd3f3cbb2f8e6ce9b44668e78ad8d4a7377 (height 205921) for 1 address
+...
+13:45:49 2015-04-13 [INF] WLLT: Finished rescan for 1 address (synced to block 0000000005cecab1013ecb1275a3e0c9623c4a497a57b6b6bf0fc1525aca1fbf, height 335146)
+```
+
+During the rescan, relevant transactions from previously unseen blocks are added
+to the wallet database and spend tracking is updated accordingly.  After the
+rescan at startup finishes, a wallet is marked in sync with the chain.
+
+When wallet is started without any transaction history, a rescan is performed
+for all blocks since the creation date of the wallet's first address.  There are
+two situations when this holds true:
+
+1. The wallet is newly created or was recreated from the seed
+
```

Comment:
I believe markdown formats lists better if there is no newline between the list items.